ocs_ci/ocs/resources/mcg.py

```diff
@@ -143,10 +143,24 @@ def __init__(self, *args, **kwargs):
             pods = pod.get_pods_having_label(label=constants.RGW_APP_LABEL, namespace=self.namespace)
```

Comment:
Can you also please change the line above from:
logger.info('Checking whether RGW pod is not present on AWS platform')

to something like:
logger.info('Checking whether RGW pod is not present.')
or mention all possible cases like not present on cloud providers, external mode?  If the storagecluster_independent_check means it's external mode :)

Reply:
done